etl/steps/data/meadow/oecd/2024-07-01/passenger_travel.py

```diff
@@ -0,0 +1,41 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+COLS_TO_KEEP = ["Reference area", "Vehicle type", "TIME_PERIOD", "OBS_VALUE"]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("passenger_travel.csv")
+
+    # Load data from snapshot.
+    tb = snap.read()
+
+    # drop columns not needed
+    tb = tb[COLS_TO_KEEP]
+
+    # Rename columns
+    tb = tb.rename(columns={"Reference area": "country", "TIME_PERIOD": "year"})
+
+    # fix dtypes and scale up data
+    tb["year"] = tb["year"].astype("Int64")
+    tb["OBS_VALUE"] = tb["OBS_VALUE"].astype("Int64") * 1_000_000
```

Comment:
It's not clear to me why you are scaling up? It's related to me not knowing about passenger-kilometers

Reply:
Oh, the data is just given in millions - I added it to the comment to make it clearer